save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/TestSuitesController.kt

```diff
@@ -72,14 +54,36 @@ class TestSuitesController(
     fun getTestSuiteById(@PathVariable id: Long) =
             ResponseEntity.status(HttpStatus.OK).body(testSuitesService.findTestSuiteById(id))
 
+    /**
+     * @param ids list of test suite ID
+     * @return response with names of test suite with id from provided list
+     */
+    @PostMapping("/internal/test-suite/names-by-ids")
+    fun findAllTestSuiteNamesByIds(@RequestBody ids: List<Long>) = ids
```

Comment:
why do we need to find multiple test suites by ids?

Reply:
it's needed for orchestrator to setup `--include-suites` on orchestrator side. there we have only Execution with testSuiteIds

Reply:
but we don't need the next method -- will remove